superset-frontend/cypress-base/cypress/integration/dashboard/markdown.test.ts

```diff
@@ -25,10 +25,6 @@ describe('Dashboard edit markdown', () => {
   });
 
   it('should load AceEditor on demand', () => {
-    let numScripts = 0;
-    cy.get('script').then(nodes => {
-      numScripts = nodes.length;
-    });
```

Comment:
nit: we should maybe update the test name to reflect the fact that we're testing the edit feature.

Reply:
Done